src/index.html

```diff
@@ -1,17 +1,200 @@
+          <label for="accept-agreement">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="accept-agreement"
+            id="accept-agreement"
+            tabindex="5"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset__opt">
+        <legend>Registration</legend>
+
+        <div class="form__input">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            tabindex="6"
+          >
+        </div>
+
+        <div class="form__input">
+          <label for="pass">Password:</label>
+          <input
+            type="password"
+            name="pass"
+            id="pass"
+            tabindex="7"
+            minlength="5"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset__opt">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form__input">
+          Do you love cats?
+          <input
+            type="radio"
+            name="lovecat"
+            id="lovecat1"
+            tabindex="8"
+          >
+          <label for="lovecat1">Yes</label>
+          <input
+            type="radio"
+            name="lovecat"
+            id="lovecat2"
+          >
+          <label for="lovecat2">No</label>
+        </div>
+
+        <div class="form__input">
+          <label for="favorite-color">What is your favorite color?</label>
+          <input
+            type="color"
+            name="favorite-color"
+            id="favorite-color"
+            tabindex="9"
+          >
+        </div>
+
+        <div class="form__input">
+          <label for="time-bed">What time do you go to bed?</label>
+          <input
+            type="time"
+            name="time-bed"
+            id="time-bed"
+          >
+        </div>
+
+        <div class="form__input">
+          <label for="favorite-car">What are your favorite brand of cars?</label>
+          <select
+            name="favorite-car"
+            id="favorite-car"
+            multiple tabindex="10"
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">LADA</option>
+          </select>
+        </div>
+
+        <div class="form__input">
+          <label for="rate-work">How do you rate our work?</label>
+          <input
+            type="range"
+            name="rate-work"
+            id="rate-work"
+            value="0"
+            tabindex="11"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset__opt">
+        <legend>Additional info:</legend>
+
+        <div class="form__input">
+          <label for="comments">Comments:</label>
+          <textarea
+          name="comments"
+          id="comments"
+          tabindex="12"
+         ></textarea>
```

Comment:
3rd item of [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
```suggestion
          <textarea
            name="comments"
            id="comments"
            tabindex="12"
         ></textarea>
```